sidebars.js

```diff
@@ -245,6 +245,7 @@ module.exports = {
             'self-hosted/manage/buildkit-high-performance',
             'self-hosted/manage/backup',
             'self-hosted/manage/custom-resource-definitions',
+            'self-hosted/manage/airgap',
```

Comment:
Can we move this under the "Complete the installation section"?

Reply:
This is pretty advanced. Maybe we can start folding it into a different section? Maybe we should have an 'advanced scenarios' section or something.   

IMO Fine-tuning buildkit, doing backups, interacting with CRDs, and doing air-gapped installations are not 'completing your installation' but  a separate work stream. If we put everything, we risk that section of the docs becoming a hodge-podge of options without a straightforward narrative.  We could even consider separating installation tasks from app-level configuration from dev-level configuration in a way that maps the personas that typically perform those tasks?

Reply:
"Complete the installation section" is to have a production ready installation, this is more than that, it only applies to some scenarios. That's why sections like airgap, backups, CRDs, ... are under the "Operating" section